### ISSUE TITLE
How to tune and use lambda in "final loss function for training a student model"?

### ISSUE THREAD

**Issue by cizhouyu** [chengang95/UnKD#2]
I'm interest in "final loss function for training a student model" in the paper, but i do not know how to use and tune the $\lambda$ in the loss function in code?
![Snipaste_2023-04-03_11-18-31](https://user-images.githubusercontent.com/60316519/229403896-6d531db0-cd01-47b5-b44d-23c2462c2a25.jpg)
